support-frontend/assets/helpers/render.js

```diff
@@ -6,7 +6,7 @@ import { logException } from 'helpers/logger';
 // Without this the build-time pre-rendering breaks, because fetch is undefined when running with node
 const safeFetch = (url: string, opts) => {
   if (typeof fetch !== 'undefined') {
-    fetch(url, opts)
+    fetch(url, opts);
```

Comment:
are you up to date with master? I think these changes are already in: https://github.com/guardian/support-frontend/blob/master/support-frontend/assets/helpers/render.js#L9